README.md

```diff
@@ -1,20 +1,20 @@
-# TerraGoat - Vulnerable Terraform Infrastructure 
+# TerraGoat - Vulnerable Terraform Infrastructure
```

Comment:
@AErmie This entire file will be overwritten by template + auto-generated part, so all the changes here are irrelevant

Reply:
Thanks, but I'm not sure how to remove the ReadMe file from the Pull Request after it' been submitted.